examples/browser/test/pages/main-page.ts

```diff
@@ -57,15 +63,102 @@ export class MainPage {
         }
     }
 
+    /* Returns true id Menu is loaded */
     public waitForLoadingMenu(): void {
         browser.waitForExist('.p-Widget.p-Menu.p-MenuBar-menu');
     }
 
+    /* Get the Location of the Tab */
     public getxBarTabPosition(tabNumber: Number): Number {
-        return browser.getLocation(`ul.p-MenuBar-content > .p-MenuBar-item:nth-child(${tabNumber}`, 'x')
+        return browser.getLocation(`ul.p-MenuBar-content > .p-MenuBar-item:nth-child(${tabNumber}`, 'x');
     }
 
+    /* Right click on directory structure to display the context menu works */
     public getxMenuPosition(): Number {
-        return this.driver.getLocation('.p-Widget.p-Menu.p-MenuBar-menu', 'x')
+        return this.driver.getLocation('.p-Widget.p-Menu.p-MenuBar-menu', 'x');
+    }
+
+    /* Test to check if the browser directory expands to shows files within it */
+    public isFolderStructureExpandable(): Boolean {
```

Comment:
I have fixed that once in #165, why do we have the `Boolean`, `Number`return types everywhere instead of the primitive ones? Am I missing something?

Reply:
I was following the same signature that was already present in some of the early test cases.

Reply:
I think that's my mistake from my first PR on theia, indeed there is no need for the wrapper types.